packages/fiber/src/core/renderer.ts

```diff
@@ -124,8 +124,8 @@ function createRenderer<TCanvas>(roots: Map<TCanvas, Root>) {
       // Event-handlers ...
       //   are functions, that
       //   start with "on", and
-      //   contain the name "Pointer", "Click", "ContextMenu", or "Wheel"
-      if (is.fun(newProps[objectKeys[i]]) && /^on(Pointer|Click|ContextMenu|Wheel)/.test(objectKeys[i])) {
+      //   contain the name "Pointer", "Click", "DoubleClick", "ContextMenu", or "Wheel"
+      if (is.fun(newProps[objectKeys[i]]) && /^on(Pointer|Click|DoubleClick|ContextMenu|Wheel)/.test(objectKeys[i])) {
```

Comment:
Would it be safer to use the keys from here instead?
https://github.com/pmndrs/react-three-fiber/blob/c2a73ee8e720d0045ff825e784b16d715f101cdb/packages/fiber/src/web/events.ts#L8-L19

Reply:
this check specifically excludes a few of them though